gneiss/regression/_mixedlm.py

```diff
@@ -145,7 +147,11 @@ def mixedlm(formula, table, metadata, groups, **kwargs):
                           **kwargs)
         submodels.append(mdf)
 
-    return LMEModel(submodels, balances=table)
+    # ugly hack to get around the statsmodels object
```

Comment:
In case you are wondering why we are doing this I raised an issue here #187.
Basically, the multidimensional response statistics in statsmodels are lacking at the moment, so we are doing some hacks here to overcome that.